functions/src/model/mapper/image-mapper.test.ts

```diff
@@ -0,0 +1,25 @@
+import * as test from 'firebase-functions-test'
```

Comment:
hey @marcosjordao , what's this image mapper for?

Reply:
The `id` field was added to the [`Image` interface](https://github.com/theaiscope/GDD-backend/pull/22/files#diff-79976c206fc733f01a763b034abe9af4882352260c860e0f3a37deacd36eafdb), because it will be required for taking actions over an Image (like Skip/Save image id X).

With that, this mapper is to map a `DocumentSnapshot` coming from the Firestore query result (`.data()`) to an `Image` object.

Please note that the `id` field is not part of the document body on Firestore, but an attribute of the `DocumentSnapshot` object.\
So the main reason for this mapper at this moment is to compose the Image with `id` + document body.

Ref: https://firebase.google.com/docs/reference/node/firebase.firestore.DocumentSnapshot#id

`document.data()`:
![image](https://user-images.githubusercontent.com/15014647/182809390-4ee08b20-8065-4f8a-8268-8e8ba1b02b9a.png)

`document.id`:
![image](https://user-images.githubusercontent.com/15014647/182809503-7f4af975-0040-4e6d-b76e-0643a6eea750.png)

`image.ts`:
https://github.com/theaiscope/GDD-backend/pull/22/files#diff-79976c206fc733f01a763b034abe9af4882352260c860e0f3a37deacd36eafdb